CONTRIBUTING.md

```diff
@@ -83,101 +100,91 @@ pytest -rs tests/integrations/flask/  # Replace "flask" with the specific integr
+   - Avoid modifying the hub, registering a new client or the like. The user drives the client, and the client owns integrations.
 
-### Versioning Policy
+   - Allow the user to disable the integration by changing the client. Check `Hub.current.get_integration(MyIntegration)` from within your signal handlers to see if your integration is still active before you do anything impactful (such as sending an event).
 
-This project follows [semver](https://semver.org/), with three additions:
+2. Write tests.
 
-- Semver says that major version `0` can include breaking changes at any time. Still, it is common practice to assume that only `0.x` releases (minor versions) can contain breaking changes while `0.x.y` releases (patch versions) are used for backwards-compatible changes (bugfixes and features). This project also follows that practice.
+   - Think about the minimum versions supported, and test each version in a separate env in `tox.ini`.
 
-- All undocumented APIs are considered internal. They are not part of this contract.
+   - Create a new folder in `tests/integrations/`, with an `__init__` file that skips the entire suite if the package is not installed.
 
-- Certain features (e.g. integrations) may be explicitly called out as "experimental" or "unstable" in the documentation. They come with their own versioning policy described in the documentation.
+3. Update package metadata.
 
-We recommend to pin your version requirements against `1.x.*` or `1.x.y`.
-Either one of the following is fine:
+   - We use `extras_require` in `setup.py` to communicate minimum version requirements for integrations. People can use this in combination with tools like Poetry or Pipenv to detect conflicts between our supported versions and their used versions programmatically.
 
-```
-sentry-sdk>=1.0.0,<2.0.0
-sentry-sdk==1.5.0
-```
+     Do not set upper-bounds on version requirements as people are often faster in adopting new versions of a web framework than we are in adding them to the test matrix or our package metadata.
 
-A major release `N` implies the previous release `N-1` will no longer receive updates. We generally do not backport bugfixes to older versions unless they are security relevant. However, feel free to ask for backports of specific commits on the bugtracker.
+4. Write the [docs](https://github.com/getsentry/sentry-docs). Follow the structure of [existing integration docs](https://docs.sentry.io/platforms/python/integrations/). Make sure to add your integration to the table in `python/integrations/index.md` (people forget to do that all the time).
 
-## Adding a new integration (checklist)
+5. Merge docs after new version has been released (auto-deploys on merge).
 
-1. Write the integration.
+6. (optional) Update data in [`sdk_updates.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/sdk_updates.py) to give users in-app suggestions to use your integration. May not be applicable or doable for all kinds of integrations.
```

Comment:
```suggestion
6. (optional) Update data in [`sdk_updates.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/sdk_updates.py) to give users in-app suggestions to use your integration. It may not be applicable or doable for all kinds of integrations.
```

Reply:
Will apply Daniel's suggestion for this one